external-crates/move/crates/move-analyzer/editors/code/CONTRIBUTING.md

```diff
@@ -32,8 +32,6 @@ Running the extension's tests from the command-line is cumbersome because the te
 3. From the pull-down menu in the Run and Debug view, select "Launch with Extension" and click the green arrow button. This will launch a new VS Code window that has the `move-analyzer` extension installed. If you open a `.move` file, it will be highlighted according to the any changes you've made in your local checkout of this repository.
 4. Once you've launched the extension, in the original VS Code window you'll see a small box with pause, refresh, and stop buttons. Clicking these allows you to quickly refresh the extension window with your latest changes, or quit the window.
 
-To test the Move TextMate grammar, open a file with a `.move` file extension and use the command palette to run the "Developer: Inspect Editor Tokens and Scopes" command. This displays a large window that shows what token is detected below the typing cursor in the VS Code window.
```

Comment:
Changes in this file are unrelated but while I was at it, I put in a few quick fixes